src/code/z_snap.c

```diff
@@ -60,10 +69,11 @@ s32 func_8013A240(PlayState* play) {
                     break;
             }
 
+            // If actor is recordable, run its validity function and record if valid
             if (seen != 0) {
                 pictoActor = (PictoActor*)actor;
-                if (pictoActor->pictoFunc != NULL) {
-                    if ((pictoActor->pictoFunc)(play, actor) == 0) {
+                if (pictoActor->validationFunc != NULL) {
+                    if ((pictoActor->validationFunc)(play, actor) == 0) {
                         count++;
```

Comment:
```suggestion
                    if ((pictoActor->validationFunc)(play, actor) == 0) {
                        validCount++;
```
Or something similar?

Reply:
Yeah, although I'm not sure I really like `valid`, it was the best I could think of. At least if they're consistent they're easy to change.